messaging/kafka-streams-reactive-messaging/src/test/java/io/quarkus/ts/messaging/kafka/reactive/streams/KafkaGratefulShutdownIT.java

```diff
@@ -44,36 +42,15 @@ public class KafkaGratefulShutdownIT {
     @KafkaContainer(vendor = KafkaVendor.STRIMZI)
     static KafkaService kafka = new KafkaService();
 
-    /**
-     * If topic is not created before the test, messages inside it will not be processed.
-     * for details and todo see https://github.com/quarkus-qe/quarkus-test-suite/issues/2016
-     */
-    private void createTopic(KafkaService service) throws InterruptedException {
-        Properties properties = new Properties();
-        properties.put("bootstrap.servers", service.getBootstrapUrl());
-        try (AdminClient client = AdminClient.create(properties)) {
-            client.createTopics(List.of(new NewTopic("slow", 1, (short) 1)));
-        }
-        Thread.sleep(1000);
-    }
-
     @QuarkusApplication(classes = { SlowTopicConsumer.class,
-            SlowTopicResource.class }, properties = "kafka.grateful.shutdown.application.properties")
+            SlowTopicResource.class, ReadinessObserver.class }, properties = "kafka.grateful.shutdown.application.properties")
     static RestService app = new RestService()
             .withProperty("kafka.bootstrap.servers", kafka::getBootstrapUrl)
             .withProperty("quarkus.kafka-streams.bootstrap-servers", kafka::getBootstrapUrl)
             .withProperty(KAFKA_LOG_PROPERTY, Level.DEBUG.getName());
 
     @Order(1)
     @Test
-    public void testConnection() throws InterruptedException {
-        createTopic(kafka);
-        app.given().post("/slow-topic/sendMessage/ave").then().statusCode(200);
-        app.logs().assertContains("ave");
-    }
-
-    @Order(2)
-    @Test
     public void shouldWaitForMessagesWhenGratefulShutdownIsEnabled() throws InterruptedException {
         givenMessagesInTopic();
         if (OS.current() == OS.WINDOWS) {
```

Comment:
do we still need to wait on Windows?

Reply:
No idea, I'd better keep it because it wasn't added because of the KRaft and I don't want to bring new flakiness while dropping other.